terraform/router/aws/proxy-protocol.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
This script enables proxy protocol in the NLB. We can't use service annotations to enable it because the in-tree AWS Load Balancer implementation doesn't support it.